src/pages/OfferNew/Compare/index.tsx

```diff
@@ -23,7 +25,13 @@ export const Compare = () => {
         <Column>
           <HeadingWrapper>
             <PreHeading>Villkor</PreHeading>
-            <HeadingBlack>Jämför gärna!</HeadingBlack>
+            <HeadingBlack>
+              Allt du förväntar dig av en hemförsäkring, plus drulle
```

Comment:
We actually have like all the text keys available, I think we're going to need to get them in use before too long 😬

Reply:
Very nice, let's go though all hard-coded copy when this is merged 😄